app/components/organisms/LitePage/mobile.js

```diff
@@ -1,4 +1,5 @@
 import React, { useState, useRef, useEffect } from 'react';
+import ReactTextTransition, { presets } from 'react-text-transition';
```

Comment:
Remove changes to this file. just merged them to the main branch. And please create a branch from the latest staging branch before starting to work.